examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py

```diff
@@ -236,97 +212,99 @@ def __lookup_basic(self, is_lookup_next, input_value, response):
                 ids.append(worker_id)
                 lookupTag = worker_id
 
-        response["result"] = {
+        result = {
             "totalCount": total_count,
             "lookupTag": lookupTag,
             "ids": ids,
         }
 
-        return response
+        return result
```

Comment:
Should the return value be a well formatted json string object like below ?
{
    "jsonrpc": "2.0",
    "id": <integer>,
    "result": {
        "totalCount":total_count
        "lookupTag":lookupTag,
        "ids": ids,
    }
}

Reply:
We should just return the `result` part, and the json-rpc library will take care of wrapping the `result` into the formatted JSON-RPC object, which will be returned as `response.data` (a well formatted json string) in L133-L139